packages/core/src/components/overlay/overlay.tsx

```diff
@@ -309,10 +309,15 @@ export class Overlay extends React.Component<IOverlayProps, IOverlayState> {
     }
 
     private handleBackdropMouseDown = (e: React.MouseEvent<HTMLDivElement>) => {
-        if (this.props.canOutsideClickClose) {
-            safeInvoke(this.props.onClose, e);
+        const { backdropProps, canOutsideClickClose, enforceFocus, onClose } = this.props;
+        if (canOutsideClickClose) {
+            safeInvoke(onClose, e);
+        }
+        if (enforceFocus) {
```

Comment:
would be super great to add a unit test for this case. we have others for `enforceFocus`, should be a matter of copy-and-edit.